src/utils/proxy.mjs

```diff
@@ -742,19 +740,6 @@ export const startProxy = async function ({
 
   const eventQueue = [once(primaryServer, 'listening')]
 
-  // If we're running the main server on HTTPS, we need to start a secondary
-  // server on HTTP for receiving passthrough requests from edge functions.
-  // This lets us run the Deno server on HTTP and avoid the complications of
```

Comment:
With this PR, we're passing the "potentially untrusted certificate" into Deno via the `--certs` flag.

This functionality was originally implemented in https://github.com/denoland/deno/pull/3972, and it works by adding the cert as a trusted root certificate to the HTTP client:

https://github.com/denoland/deno/pull/3972/files#diff-5c6fb275a66a1b02f99597f6e9b36297274abd4b16926b9fb2d412f94ee0700eR52

From my perspective, this should be fine. Maybe Eduardo can elaborate on the complications they had in mind when they're back.

Reply:
Have you seen the PR that introduced that functionality? https://github.com/netlify/cli/pull/5409